packages/dds/matrix/src/serialization.ts

```diff
@@ -3,10 +3,10 @@
  * Licensed under the MIT License.
  */
 
+import { fromBase64ToUtf8 } from "@fluidframework/common-utils";
```

Comment:
There should be almost no reference to fromBase64ToUtf8() or read() in runtime code after your changes.